great_expectations/checkpoint/checkpoint.py

```diff
@@ -34,7 +34,7 @@
 )
 from great_expectations.compatibility.typing_extensions import override
 from great_expectations.core.expectation_validation_result import (
-    ExpectationSuiteValidationResult,  # noqa: TCH001
+    ExpectationSuiteValidationResult,
```

Comment:
a positive side effect of updating the pydantic runtime base classes in pyproject.toml